pages/04.Reference/07.dns/docs.en.md

```diff
@@ -0,0 +1,72 @@
+
+After creating zones in the SysEleven Stack DNS service you can see the responsible authoritative name servers by listing the recordsets in the horizon dashboard or by running `openstack recordset list example.com.`:
+
+```shell
+$ openstack recordset list example.com.
++--------------------------------------+--------------------------+-------+--------------------------------------------------------------------------------+--------+--------+
+| id                                   | name                     | type  | records                                                                        | status | action |
++--------------------------------------+--------------------------+-------+--------------------------------------------------------------------------------+--------+--------+
+| 53fee216-577a-41b4-9d48-025239b38411 | example.com.             | NS    | ns01.cloud.syseleven.net.                                                      | ACTIVE | NONE   |
+|                                      |                          |       | ns02.cloud.syseleven.net.                                                      |        |        |
+|                                      |                          |       | ns04.cloud.syseleven.net.                                                      |        |        |
+|                                      |                          |       | ns03.cloud.syseleven.net.                                                      |        |        |
++--------------------------------------+--------------------------+-------+--------------------------------------------------------------------------------+--------+--------+
+```
+
+!! Please note that DNS zones are always represented with a trailing dot in our DNS service, so if your domain name is `example.com`, the corresponding zone name would be `example.com.`
+
+### Zone transfer to different projects
+
+It is possible to transfer zones If they need to be maintained using different OpenStack projects. For example you can have a zone named `company.com.` in `project-a` and another zone named `team-1.company.com.` in `project-b`.
+
+#### Zone disputes
+
+It is not automatically possible for us to verify ownership of your domain names, so in very rare circumstances your domain name might have been claimed by another customer already. Please contact our customer support in this case.
+
+### Secondary Zones
+
+It is possible to use our authoritative name server infrastructure together with a hidden master that you maintain yourself. You can accomplish that by creating a secondary zone. Using the OpenStack CLI you can create it like this: `openstack zone create --type secondary --masters master-1.example.com. master-2.example.com`.
+
+Our authoritative name servers will respect the `Refresh`, `Retry`, `Expire` and `TTL` timings of the configured DNS master and our servers will poll for updates using the `AXFR` protocol accordingly.
+
```

Comment:
Let's state from which ip addresses customer should expect AXFR queries to come (currently all 4 public dns resolvers)

Reply:
@rnurgaliyev  fixed in https://github.com/syseleven/grav-docs-syseleven-stack/pull/41/commits/91fb71fd6736ef90249aed555617e6bfadc7881a